nuxeo-core/nuxeo-core-convert-plugins/src/main/java/org/nuxeo/ecm/core/convert/plugins/text/extractors/XML2TextConverter.java

```diff
@@ -39,18 +39,18 @@
 
     @Override
     public BlobHolder convert(BlobHolder holder, Map<String, Serializable> parameters) throws ConversionException {
-        return new SimpleBlobHolder(new StringBlob(convert(holder.getBlob(), parameters)));
+        return new SimpleBlobHolder(new StringBlob(convert(holder.getBlob())));
     }
 
-    String convert(Blob blob, Map<String, Serializable> parameters) {
```

Comment:
Please, do not change method signature for backward compatibility, deprecate it if needed and add a new one.

Reply:
Yes, i removed because the parameter is never used and the correction will be just for 11.1 that why i didn't care more about compatibility. I will add the new method and deprecated the original one :)